### ISSUE TITLE
Personal audit revealed missing css rules in current distribution

### ISSUE THREAD

**Issue by jeromystewart** [twbs/bootstrap#16065]
Hey guys, great work on bootstrap

I'm in the process of building a php version of less and I am using bootstrap for my base (testing & analysis).

I am unable to find the final rules rendered by the statement:
 (location: navbar.less) - compressed

  .navbar-collapse,
  .navbar-form {
    border-color: @navbar-default-border;
  }

in your current distribution. This could be due to some superseding rule or it could be that less is not rendering it. Neverthe(less) no pun intended, it seems to be missing.

To put it another way, I should see:

.navbar-collapse,.navbar-form or .navbar-form,.navbar-collapse
somewhere in the bootstrap stylesheet but I do not.

I also looked for .navbar-collapse and .navbar-form anywhere in the document where one of the style attributes/properties was border-color

Although I am about 4 IPA's deep, I have checked a half dozen times and within the margin of (beer) error I think there is a problem somewhere

If I am in error, I do apologize in advance

**Comment by kkirsche** [comment]
I'm not sure I understand. Is this supposed to be the rule for the border color of the collapsed navbar button with the three bars? Thanks in advance for the extra clarification.

**Comment by jeromystewart** [comment]
In truth, I don't know exactly what the rule does, I found out about this by auditing my own rendering of bootstrap against the "less" rendering of bootstrap

**Comment by jeromystewart** [comment]
I mean it's relatively simple, I am not seeing the "less" output for this rule in your distribution

**Comment by jeromystewart** [comment]
Also I am referenceing http://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css

**Comment by patrickhlauke** [comment]
```
.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}
```
